index.html

```diff
@@ -8,6 +8,198 @@
     <title>Technical</title>
 </head>
 <body class="body">
-   
-</body>
+  <section>
+    <nav class="menu">
+           <ul>
+                <li class="list-link"><h1 class="header-menu">JS Documentation</h1></li> 
+                 <li class="list-link"><a class="menu-link" href="#introduction">Introduction</a></li>
+                 <li class="list-link"><a class="menu-link" href="#what-you-should-already-know">What you should already know</a></li>
+                 <li class="list-link"><a class="menu-link" href="#javascript-and-java">JavaScript and Java</a></li>
+                 <li class="list-link"><a class="menu-link" href="#hello-world">Hello world</a></li>
+                 <li class="list-link"><a class="menu-link" href="#variables">Variables</a></li>
+                 <li class="list-link"><a class="menu-link" href="#declaring-variables">Declaring variables</a></li>
+                 <li class="list-link"><a class="menu-link" href="#variable-scope">Variable scope</a></li>
+                 <li class="list-link"><a class="menu-link" href="#global-variables">Global variables</a></li>
+                 <li class="list-link"><a class="menu-link" href="#constants">Constants</a></li>
+                 <li class="list-link"><a class="menu-link" href="#data-types">Data types</a></li>
+                 <li class="list-link"><a class="menu-link" href="#if-else-statement">if...else statement</a></li>
+                 <li class="list-link"><a class="menu-link" href="#while-statement">while statement</a></li>
+                 <li class="list-link"><a class="menu-link" href="#function-declarations">Function declarations</a></li>
+                 <li class="list-ref"><a class="menu-link" href="#reference">Reference</a></li>
+       </ul>  
+    </nav>          
+   </section>
+   <section class="container">
+    <div id="introduction">
```

Comment:
change to sections. This applies to all areas of the documentation. Ids point to sections